addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -43,6 +43,10 @@
         max-width: unset;
         width: 100%;
         padding: 0;
+
+        .o_ChatterTopbar_tools_border, .o_ChatterTopbar_tools .o_ChatterTopbar_center {
```

Comment:
This level of specificity  should be unnecessary, have you tried defining `--ChatterTopbar-border` value into `o_FormRenderer_chatterContainer` directly?

Reply:
Indeed it, thank you